src/test/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFDataUnitTest.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import htsjdk.samtools.util.Interval;
+import org.apache.commons.lang.math.IntRange;
+import org.broadinstitute.hellbender.tools.exome.AllelicCount;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by davidben on 11/17/15.
+ */
+// TODO: expand this, move initialization test to AFInitializerUnitTest
+public class AFDataUnitTest {
+
+    @Test
+    public void testGetters() {
```

Comment:
I test them out of paranoia that Travis will complain about coverage going down.  @LeeTL1220 do we have a policy?